tests/test_pandas_extension.py

```diff
@@ -382,6 +369,8 @@ def test_combine_le(self, data_repeated):
         # Test that combine works when doing a <= (le) comparison
         # Fletcher returns 'fletcher_chunked[bool]' instead of np.bool as dtype
         orig_data1, orig_data2 = data_repeated(2)
+        if pa.types.is_list(orig_data1.dtype.arrow_dtype):
```

Comment:
Can this be covered by some kind of fixture? Adding it to every test is a lot of boilerplate

Reply:
Implemented a marker for skipping/xfailing tests.